src/utilities/params.ts

```diff
@@ -7,6 +7,19 @@ const extras: ParamExtras = {
   },
 }
 
+const stringParam: ParamGetSet<unknown> = {
+  get: (value) => {
+    return value
+  },
+  set: (value, { invalid }) => {
+    if (typeof value !== 'string') {
+      throw invalid()
+    }
+
+    return value.toString()
```

Comment:
we probably don't need the toString() since we've already checked if its not a string right?